Clarify how to restore a database backup in cluster (#3035)
neo4j/docs-operations: modules/ROOT/pages/backup-restore/restore-backup.adoc

[[restore-backup]]
= Restore a database backup

Neo4j suggests a few options to restore a database backup artifact (full or differential) created with the xref:backup-restore/online-backup.adoc[`neo4j-admin database backup`] command.

== Restore a database on a standalone server

To restore a database from a backup in a standalone server environment, use one of the following methods:

* Use the <<restore-backup-command, `neo4j-admin database restore`>> command.
* Restore a database using URI approach.
See xref:database-administration/standard-databases/seed-from-uri.adoc[] for detailed information.
* Recreate a database using the xref:database-administration/standard-databases/recreate-database.adoc[`dbms.recreateDatabase()` procedure].

> * Use the xref:backup-restore/copy-database.adoc[`neo4j-admin database copy`] command to restore database data from a backup.

Author: ```suggestion
* label:new[Introduced in 2026.03] Use the xref:backup-restore/copy-database.adoc[`neo4j-admin database copy`] command to restore database data from a backup.
```

======
=====

=== Restore a database backup in a cluster

> To restore a database backup in a cluster, you can use one of the following methods:

Author: ```suggestion
To restore a database in a clustered environment from a backup, you can use one of the following methods:
```

> To restore a database in a clustered environment from a backup, you can use one of the following methods:

Author: ```suggestion
To restore a database from a backup in a clustered environment, you can use one of the following methods:
```

* Designate one of the servers to be used as a seeder, and restore the database backup on that server.
For more information, see xref:clustering/databases.adoc#cluster-designated-seeder[Restore a database using a server as a designated seeder].

> * Recreate a database from a backup or server URI.

Reviewer: I would also suggest putting this first as it is our preferred way of doing things.

> See xref:clustering/databases.adoc#restore-database-using-uri-approach[Restore a database using URI approach] for more information.

Reviewer: I would drop 'backup or server' from this sentence as we refer to all the various URI just as the URI approach.

* Recreate a database using the `dbms.recreateDatabase()` procedure.
See xref:clustering/databases.adoc#restore-database-using-recreate-procedure[Restore a database using `dbms.recreateDatabase()`] for more details.
* label:new[Introduced in 2026.03] Use the xref:backup-restore/copy-database.adoc[`neo4j-admin database copy`] command.
See xref:clustering/databases.adoc#restore-database-using-copy-command[Restore a database backup using the `neo4j-admin database copy` command] for more details.

=== Restore users and roles metadata

> Roles and users metadata are backed up during the backup process by default.

Author: ```suggestion
Users, roles, and auth rules metadata are backed up during the backup process by default.
```
